src/core/stream/period/period_stream.ts

```diff
@@ -219,6 +223,7 @@ export default function PeriodStream({
 
           return segmentBuffersStore.waitForUsableBuffers().pipe(mergeMap(() => {
             return observableConcat(cleanBuffer$,
+                                    needsBufferFlush$,
```

Comment:
Isn't `cleanBuffer` equal to `EMPTY` when strategy.type is not equal to `"clean-buffer"`?

If so how does it work: we seem to be just seeking without removing anything, no?

Reply:
Ho, I thought it was by just seeking that the `videoElement` will erase buffer, so I was not understand why a "seek" would flush the buffer but it makes sense now :)